scripts/basket.js

```diff
@@ -0,0 +1,58 @@
+class Basket {
+    constructor() {
+        this.items = [];
+        this.totalValue = 0;
+    }
+
+    clear() {
+        this.items.length = 0;
```

Comment:
IMO, lepiej resetować tablice tak `this.items = []`

Reply:
Poprawione.